app/controllers/admin_dashboard_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AdminDashboardController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Can non-admin users access this dashboard? Do we need an extra check saying 'if user not admin, kick them out'?

Reply:
Probably worth adding a test describing this behaviour too.

Reply:
@kicferk1 that did cross my mind but I felt it was out of scope for this login pr. I'd typically use pundit for that and add policies and policy tests. I won't add them to this pr but I can do a separate one after this on when we beef up this controller.

Reply:
Alright, happy for it to be a separate PR